testsuite/ui/views/admin/audience/account_user.py

```diff
@@ -55,3 +60,18 @@ def prerequisite(self):
     def is_displayed(self):
         return BaseAudienceView.is_displayed.fget(self) and self.path in self.browser.url \
                and self.username.is_displayed and self.email.is_displayed
+
+
+class AccountUserDetailView(BaseAudienceView):
+    """View representation of Accounts User page"""
+    path_pattern = 'buyers/accounts/{account_id}/users/{user_id}'
+
+    def __init__(self, parent, account, user):
+        super().__init__(parent, account_id=account.entity_id, user_id=user.entity_id)
+
+    def prerequisite(self):
+        return AccountsDetailView
+
+    @property
+    def is_displayed(self):
+        return BaseAudienceView.is_displayed.fget(self) and self.path in self.browser.url
```

Comment:
Please specify another element that can identify this page. The current is_displayed method can match multiple pages.